include/deal.II/base/thread_management.h

```diff
@@ -64,55 +64,21 @@ DEAL_II_NAMESPACE_OPEN
 namespace Threads
 {
   /**
-   * A class implementing a <a
-   * href="https://en.wikipedia.org/wiki/Lock_(computer_science)">mutex</a>.
-   * Mutexes are used to lock data structures to ensure that only a
-   * single thread of execution can access them at the same time.
-   *
-   *
-   * <h3>Differences to std::mutex and copy semantics</h3>
-   *
-   * This class is like `std::mutex` in almost every regard and in
-   * fact is derived from `std::mutex`. The only difference is that
-   * this class is copyable when `std::mutex` is not.  Indeed, when
-   * copied, the receiving object does not copy any state from the
-   * object being copied, i.e. an entirely new mutex is created. These
-   * semantics are consistent with the common use case if a mutex is
-   * used as a member variable to lock the other member variables of a
-   * class: in that case, the mutex of the copied-to object should
-   * only guard the members of the copied-to object, not the members
-   * of both the copied-to and copied-from object. Since at the time
-   * when the class is copied, the destination's member variable is
-   * not used yet, its corresponding mutex should also remain in its
-   * original state.
-   *
-   * @author Wolfgang Bangerth, 2002, 2003, 2009
+   * This class is a thin wrapper around `std::mutex`. The only difference
```

Comment:
Why replace the more descriptive documentation?

Reply:
I have kept almost all of the descriptive documentation in place and only removed the parts that will be factually incorrect after this pull request.

Reply:
If you want, I can put the `@author` clause back in place. I did not mean to offend you.

Reply:
No, I don't actually care about the `@author` tag at all :-) I cared about the link to the wikipedia page, which I think gives useful background information...

Reply:
Ah! Let me put it back.